index.markdown

```diff
@@ -28,7 +28,7 @@ graphics:
       src: /assets/img/circles/cpu.png
       alt: "CPU icon"
     title: Contribute
-    description: Everyone is welcome. [Learn how you can contribute](/join) to CivicActions Jekyll Vanilla site.
+    description: Everyone is welcome. [Learn how you can contribute](/ca-jekyll-vanilla/join) to CivicActions Jekyll Vanilla site.
```

Comment:
Good catch @TheBoatyMcBoatFace. Can we make this maybe relative so it can work with either URLs instead of the hardcoded one? This change is causing the link checker to fail.